.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: enhancement
+assignees: ''
+
+---
+
+## Purpose
+What will this feature provide?
+
+### Is your feature request related to a problem? Please describe.
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+
+## Proposed Solution
+If you have an idea on how to implement this feature, please share!
+
+###Describe alternatives you've considered (optional)
```

Comment:
Looks like there's a space missing between `###` and the text.

Reply:
fixed!